### PR TITLE
Fix python version restrictions in tox.ini definitions

### PR DESCRIPTION
**Pull request by kaypee90**
# Description

This change updates tox to reduce python version restrictions in tox.ini definitions

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #205 

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -21,7 +21,7 @@ commands_pre =
     pip3 install -e {toxinidir}/ext/dapr-ext-fastapi/
 
 [testenv:flake8]
-basepython = python3.7
+basepython = python
```
Comment (wcs1only) [inline tox.ini:24]: Did you mean python3 here?

Comment (kaypee90) [reply tox.ini]: Yes, changes made

**Review by wcs1only — APPROVED** [review]

**Comment by wcs1only** [comment]
Nice work, thanks! Your branch is out-of-date and I don't have permission to merge to your branch. Could you please merge from latest master?

**Comment by kaypee90** [comment]
> Nice work, thanks! Your branch is out-of-date and I don't have permission to merge to your branch. Could you please merge from latest master?

The branch has been updated

**Pull request merged**